pkg/proxy/iptables/proxier.go

```diff
@@ -1335,25 +1340,19 @@ func (proxier *Proxier) syncProxyRules() {
 						"-A", string(svcChain),
 						args,
 						"-j", string(KubeMarkMasqChain))
-				} else {
-					// TODO: Make all nodePorts jump to the firewall chain.
-					// Currently we only create it for loadbalancers (#33586).
-
-					// Fix localhost martian source error
-					loopback := "127.0.0.0/8"
-					if isIPv6 {
-						loopback = "::1/128"
```

Comment:
Is this no longer necessary for IPv6?

Reply:
Also explained in the commit message: "But remove the ::1 case since that didn't
actually work anyway." (Someone added the IPv6 case when doing the initial IPv6 porting, but we discovered later that the kernel doesn't let you NAT a packet that was sent to ::1.)